packages/account/src/Sections/Security/TwoFactorAuthentication/digit-form.tsx

```diff
@@ -1,57 +1,69 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Formik, Form, Field } from 'formik';
+import { Formik, Form, Field, FormikProps, FormikHelpers, FieldProps } from 'formik';
 import { Input, Button } from '@deriv/components';
 import { getPropertyValue, WS } from '@deriv/shared';
 import { localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+
+type TResponse = {
+    error?: {
+        message: string;
+        code: string;
+    };
+};
+
+type TDigitFormValues = {
+    digit_code: string;
+};
+
+const DigitForm = observer(() => {
```

Comment:
its giving a code smell for the complexity. you might need to refactor the condition blocks